docs/setting-up-your-first-cluster.md

```diff
@@ -545,7 +545,7 @@ Create a namespace:
 kubectl create namespace dev
 ```
 
-Create an nginx deployment and expose it within the cluster:
+Create a nginx deployment and expose it within the cluster:
```

Comment:
The original seems good for me.

Reply:
As far as I can see, "an nginx" is more used, even on the nginx website
https://docs.nginx.com/nginx/deployment-guides/setting-up-nginx-demo-environment/

Reply:
So I would back @oomichi there and keep it as "an nginx" @Fish-pro